hq_superset/oauth.py

```diff
@@ -56,6 +56,15 @@ def set_oauth_session(self, provider, oauth_response):
         # }
         session[SESSION_OAUTH_RESPONSE_KEY] = oauth_response
 
+    def set_role_permissions(self, role, permissions):
```

Comment:
I was also thinking it feels a little out of place, but then all the permissions-related functionality sits in the class this class inherits from, and since this is a permissions-related method I thought it's probably best putting it here. Any other suggestions are welcome.